src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -0,0 +1,22 @@
+<template>
+    <article class="slds-setup-assistant__step">
+        <c-accordion-section>
+            <div slot="clickableHeaderSection"
+                class="slds-m-left_medium">
+                <c-gs-accordion-label
+                    number={group.id}
+                    title={group.title}>
+                </c-gs-accordion-label>
+            </div>
+            <div slot="extraInfoSection">
+                <p class="slds-m-left_x-large slds-p-left_large">{group.desc}</p>
+            </div>
+            <div class="slds-m-left_x-large">
+                <c-gs-accordion-section open="true">
+                    <p style="margin:3rem;height: 10rem;">Items</p>
```

Comment:
Should "Items" in this and the next element be custom labels vs. hardcoded text?

Reply:
This is a placeholder for now. Will be replaced in another pr